.github/workflows/release-please.yml

```diff
@@ -48,6 +55,6 @@ jobs:
         uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
-          SLACK_MESSAGE: Successfully published SKY UX ${{ steps.vars.outputs.tag }} to NPM. http://github.com/${{ github.repository }}/blob/main/CHANGELOG.md
+          SLACK_MESSAGE: Successfully published SKY UX ${{ steps.release.outputs.tagName }} to NPM. http://github.com/${{ github.repository }}/blob/main/CHANGELOG.md
```

Comment:
The output is named `tag_name`. [Docs](https://github.com/google-github-actions/release-please-action#outputs).

Reply:
They've aliased their outputs to support camel case, but I'll change it so we can get this out.
https://github.com/google-github-actions/release-please-action/blob/main/index.js#L230